app/graphql/types/comment_type.rb

```diff
@@ -0,0 +1,16 @@
+module Types
+  class CommentType < Types::BaseObject
+    field :ancestry, String, null: true
```

Comment:
@tonekk My bad :pray:. I read the [field documentation](https://graphql-ruby.org/fields/introduction.html#field-return-type) and since it used examples without `null` and said that `true` was the default value, I assumed it would work.

But I've just tested it and it doesn't, so feel free to ignore this comment.

Reply:
All good :)